### PR TITLE
Added CI

### LINKED ISSUE #18: Add CI/CI
Setup github actions for continuous builds and verifying that the new PR does not break the build.

### PR DESCRIPTION
**Pull request by RaisinTen**
fixes #18
this workflow is supposed to build `kernel.iso` on `ubuntu-latest` on every `push` on every `branch`

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '*'
```
Comment (junaidrahim) [inline .github/workflows/ci.yml:6]: Change this to `master` please. We only want to make sure `master` does not break.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  pull_request:
+    branches:
+      - '*'
```
Comment (junaidrahim) [inline .github/workflows/ci.yml:9]: Change this to `master` please. We only want to make sure `master` does not break.

**Comment by RaisinTen** [comment]
@junaidrahim done! :slightly_smiling_face:

**Comment by junaidrahim** [comment]
Cool, thanks :+1:

**Pull request merged**